Bottom: bb96b1bfd624755ced69cdb7a7ccb8b1e2333f73
Top:    39cd5f9007761bb378cdefc26bfab95ee177c2b5
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-03-23 00:10:48 +0000

lesstif: Use Coord type for line-width, not int


---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index ed10b50..79a914a 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -52,7 +52,7 @@ typedef struct lesstif_gc_struct
 
   Pixel color;
   const char *colorname;
-  int width;
+  Coord width;
   EndCapStyle cap;
   char xor_set;
   char erase;
